src/js/background.js

```diff
@@ -765,6 +765,35 @@ Badger.prototype = {
     chrome.browserAction.setTitle({tabId: tab.id, title: "Privacy Badger"});
   },
 
+  debugSite: function() {
+    let self = this;
+    chrome.tabs.query({'active': true}, (tabs) => {
+      let tab = tabs[0];
+      console.log('using tab with url: ' + tab.url);
+      let browser = window.navigator.userAgent;
+      let version = chrome.runtime.getManifest().version;
+      let fqdn = window.extractHostFromURL(tab.url);
+      /* origin info:
```

Comment:
This comment seems redundant; I suggest nicely formatting the actual data structure below instead.